src/main.js

```diff
@@ -20,6 +20,9 @@ app.use(vuetify);
 // Add vuex store to vue
 app.use(store);
 
+// Use language from Local Storage if it exists
+i18n.global.locale.value = getLocalStorage('preferredLocale') ?? 'en';
```

Comment:
Per @seriouslysean's point let's fold this config into `@/i18n.js` itself on line 20 where we initialize the selected language. Then we can remove this line and the import on line 8 here